tljh-voila-gallery/tljh_voila_gallery/__init__.py

```diff
@@ -59,6 +53,7 @@ def login_url(self, base_url):
 
 @hookimpl
 def tljh_custom_jupyterhub_config(c):
+    ensure_builder_units()
```

Comment:
I just realized this is the wrong place for this. I meant https://github.com/jupyterhub/the-littlest-jupyterhub/blob/master/tljh/hooks.py#L52, which we don't use right now. Should move to a more explicit post-install hook though.

Reply:
Agreed. The best is indeed to have a dedicated post install hook (will submit a PR for that soon!).

Reply:
PR to add the `tljh_post_install` hook to TLJH: https://github.com/jupyterhub/the-littlest-jupyterhub/pull/389